English_transcripts/school/Epic Quest/sharedAssets/C1_INTRO_EQ_English.txt

```diff
@@ -0,0 +1 @@
+Hello how much of you learn using your tablet? Use your new skills to get through the gates on this fun loving journey. we call it the epic quest. At the end of the road there will be a village that you can visit and explore. Look for clues and collect objects and animals along the way. You will be able to add those items and use those clues to unlock the mystries in the village. Keep travelling on the road to unlock the more clues and visit the village often. Good luck. 
```

Comment:
Use proper punctuations. If required, do a grammar check in Word

Hello! how much **did** you learn using your tablet? Use your new skills to get through the gates **of** this fun loving journey. we call it the epic quest. At the end of the road**,** there will be a village that you can visit and explore. Look for clues and collect objects and animals along the way. You will be able to add those items and use those clues to unlock the **mysteries** in the village. Keep travelling on the road to unlock ~~the~~ more clues and visit the village often. Good luck.

Reply:
Updated the transcript.